Library/Homebrew/env_config.rb

```diff
@@ -162,6 +162,10 @@ module EnvConfig
                      "of Ruby is new enough.",
         boolean:     true,
       },
+      HOMEBREW_GITHUB_API_URL:                   {
+        description: "Use this URL for accessing the GitHub API, such as https://git.example.com/api/v3" \
+                     "for a GitHub Enterprise Server. ",
+      },
```

Comment:
I think this needs to be more specific as to where it's used. It's not using this instead for e.g. all GitHub searches.